tb_plugin/torch_tb_profiler/plugin.py

```diff
@@ -121,8 +122,13 @@ def views_route(self, request):
         name = request.args.get("run")
         worker = request.args.get("worker")
         run = self._get_run(name)
-        profile = run.get_profile(worker)
-        views = sorted(profile.views, key=lambda x: x.id)
+        worker = run.workers[0]
```

Comment:
worker has been received from 'worker = request.args.get("worker")' in line123 ?

Reply:
Or this function can modify to 'profile = self._get_profile_for_request(request); views = sorted(profile.views,...);'